evap/staff/templates/staff_semester_view_course.html

```diff
@@ -6,8 +6,11 @@
 <td>
     {% if request.user.is_staff and not info_only and not semester.is_archived %}
         {% if state == 'new' or state == 'prepared' or state == 'editor_approved' or state == 'approved' or state == 'reviewed' or state == 'published' %}
-            <input type="checkbox" name="course" id="course{{ course.id }}" value="{{ course.id }}" data-toggle="tooltip"
-                data-placement="top" title="{% trans 'Click to select for course operation' %}" />
+            <div class="form-check">
+                <input class="form-check-input" type="checkbox" name="course" id="course{{ course.id }}" value="{{ course.id }}" data-toggle="tooltip"
```

Comment:
tooltip into the div to be consistent with contribution_formset.html?

Reply:
check